### PR TITLE
Improve the English of the alternatives

### LINKED ISSUE #219: Improve English of the comparison page
https://doc.bblf.sh/alternatives.html contains multiple typos and general English mistakes which should be fixed. Tagging @campoy fwiw.

### PR DESCRIPTION
**Pull request by vmarkovtsev**
Fixes #219

### CODE REVIEW

File: alternatives.md [hunk 1]
```diff
@@ -21,132 +21,134 @@ From official [Kythe documentation](https://kythe.io/docs/kythe-overview.html#_g
...
 links to declarations, resolved references, etc.
 
 Babelfish provides a unified IR across languages, which is defined
 as a non-goal of Kythe.
 
-At the same time, since Babelfish only processes one file at the time, it cannot
-resolve symbolic references in the code, while the goal on Kythe project is to
-provide this information with a very high accuracy.
+At the same time, since Babelfish only processes one file at a time, it cannot
+resolve symbolic references in the code, while the goal of Kythe project is to
```
Comment (juanjux) [inline alternatives.md]: s/of/of the

Comment (juanjux) [inline alternatives.md]:
```suggestion
resolve symbolic references in the code, while the goal of the Kythe project is to
```

**Comment by vmarkovtsev** [comment]
@juanjux Done

**Comment by juanjux** [comment]
Thanks for the PR!

**Pull request merged**